src/Websites/Websites/help/Remove-AzWebAppManagedCertificate.md

```diff
@@ -0,0 +1,138 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Websites.dll-Help.xml
+Module Name: Az.Websites
+online version:https://docs.microsoft.com/en-us/powershell/module/az.websites/remove-azwebappmanagedcertificate
+schema: 2.0.0
+---
+
+# Remove-AzWebAppManagedCertificate
+
+## SYNOPSIS
+Creates an App service managed certificate for an Azure Web App. 
+
+## SYNTAX
+
+```
+Remove-AzWebAppManagedCertificate [-ResourceGroupName] <String> [-WebAppName] <String> [[-Slot] <String>]
```

Comment:
This should be RemoveCertificate & not ManagedCeriticate

Reply:
Don't need WebappName & hostname to remove a cert, since Thumbprint is given. Refer to this https://docs.microsoft.com/en-us/cli/azure/webapp/config/ssl?view=azure-cli-latest#az_webapp_config_ssl_delete

Reply:
> This should be RemoveCertificate & not ManagedCeriticate

Given as per design review. Will change as per your suggestion.

Reply:
> Don't need WebappName & hostname to remove a cert, since Thumbprint is given. Refer to this https://docs.microsoft.com/en-us/cli/azure/webapp/config/ssl?view=azure-cli-latest#az_webapp_config_ssl_delete

Hi Sisira, in .Net SDK those are required to make the changes to host configuration. please refer to below link https://docs.microsoft.com/en-us/rest/api/appservice/webapps/createorupdatehostnamebinding